src/Test/config.jl

```diff
@@ -12,10 +12,10 @@ struct TestConfig{T <: Number}
     optimal_status::MOI.TerminationStatusCode
     basis::Bool # can get variable and constraint basis status
     function TestConfig{T}(;
-        atol::Float64 = 1e-8, rtol::Float64 = 1e-8, solve::Bool = true,
+        atol::Real = sqrt(eps(T)), rtol::Real = sqrt(eps(T)), solve::Bool = true,
```

Comment:
Isn't eps only defined for AbstractFloat?

Reply:
yep, OK `Base.rtoldefault` it is

Reply:
I think `Base.rtoldefault(T)` is OK instead of `Base.rtoldefault(one(T), one(T), 0)`, just looking at https://github.com/JuliaLang/julia/blob/e58bc723ed7b1c10da28f55385c243f6df5499cf/base/floatfuncs.jl#L299